src/components/AdoptMyShop/AdoptMyShop.scss

```diff
@@ -0,0 +1,117 @@
+@import "./../../css/variables.scss";
+
+section {
```

Comment:
Add class "adopt-my-shop" instead of section

Reply:
done